src/main/java/org/isf/admission/manager/AdmissionBrowserManager.java

```diff
@@ -47,6 +52,13 @@ public class AdmissionBrowserManager {
 
 	@Autowired
 	private AdmissionIoOperations ioOperations;
+	
+	@Autowired
+	private FileSystemPatientPhotoManager fileSystemPatientPhotoManager;
+	
+	private static final Logger LOGGER = LoggerFactory.getLogger(AdmissionBrowserManager.class);
+	
+	private static final String PATIENT_PHOTO_FROM_DATABASE = "DB";
```

Comment:
why not make this PUBLIC in a class like PatientBrowserManager and then reference it in other classes instead of replicating it and making it private?